### PR TITLE
[ng] ngio toh-5 sync and other page updates

### PR DESCRIPTION
**Pull request by chalin**

### CODE REVIEW

File: gulp/ngio-get.js [hunk 1]
```diff
@@ -136,7 +141,7 @@ module.exports = function (gulp, plugins, config) {
   gulp.task('_get-extra-dart', () => {
     const baseDir = path.join(angulario, 'public/docs/dart/latest');
     return gulp.src([
-      `${baseDir}/_util-fns.jade`,
+      `!${baseDir}/_util-fns.jade`, // 2017-03-24 stop syncing _util-fns.jade
```
Comment (kwalrath) [inline gulp/ngio-get.js]: Just curious, why is this `!` instead of `//`?

Comment (kwalrath) [inline gulp/ngio-get.js]: (since the next line uses `//`)

Comment (chalin) [inline gulp/ngio-get.js]: Commenting it out would actually have been ok too (and probably better).

Comment (chalin) [inline gulp/ngio-get.js]: Fixed

**Pull request merged**